soundata/annotations.py

```diff
@@ -36,21 +62,40 @@ class Events(Annotation):
             (as floats) in seconds in the form [start_time, end_time]
             with positive time stamps and end_time >= start_time.
         labels (list): list of event labels (as strings)
-        confidence (np.ndarray or None): array of float confidence values
-            in the range [0, 1]
+        confidence (np.ndarray or None): array of confidence values
+        labels_unit (str): labels unit, one of LABELS_UNITS
+        confidence_unit (str): confidence unit, one of CONFIDENCE_UNITS
+        intervals_unit (str): intervals unit, one of TIME_UNITS
+
+
     """
 
-    def __init__(self, intervals, labels, confidence=None) -> None:
+    def __init__(
+        self,
+        intervals,
+        labels,
+        confidence=None,
+        intervals_unit="s",
+        labels_unit="open",
+        confidence_unit="binary",
```

Comment:
Yeah I agree.. If we enforce it then we have to edit all the loaders. I'm ok doing that because we don't have as many loaders yet, but in the future it would be nice to think of a way of introducing changes without having to manually edit all files. I haven't really found a workaround for it..

Reply:
Yes, you raise a good point (about updating loaders). But hopefully the API (especially for mandatory fields) doesn't change very often (changing mandatory fields would be a breaking change so ideally we avoid it as much as possible), so I don't anticipate us having to make these updates very often at least.

If you're OK with it, I would vote for making these mandatory.